.vscode/tasks.json

```diff
@@ -0,0 +1,58 @@
+{
+  // See https://go.microsoft.com/fwlink/?LinkId=733558
+  // for the documentation about the tasks.json format
+  "version": "2.0.0",
+  "tasks": [
+    {
+      "label": "Karma Tests",
+      "type": "npm",
+      "script": "start",
+      "options": {
+        "env": {
+          "BROWSERS": "ChromeHeadless"
```

Comment:
Can we do FirefoxHeadless, just because we love it 😁

I guess the change would require some format changes in problemMatcher so I'm still okay with this.

Reply:
I wanted to, but FirefoxHeadless didn't even start karma server for me. So, I decided to be lazy. Can you check if it works locally for you?

Reply:
It seems `npm start -- --browsers Firefox` is also broken 🤔

Reply:
FirefoxHeadless works fine in CI though :thinking:

Reply:
Saving a file while in a `npm start` session starts Firefox for me. _Kinda works_